### ISSUE TITLE
Create a real ReadMe.md

### ISSUE THREAD

**Issue by jimmykane** [jimmykane/quantified-self#46]

**Comment by ryancornia** [comment]
This would be really useful.I was trying to install self hosted, but it appears there is another service involved on port 5000 (to download data from garmin)? Any direction on where to get that service and set it up?

**Comment by jimmykane** [comment]
If you are running it on localhost: 

```
http://localhost:4200/
```

Should be the port the AngularCli sets. 

If you can paste some output I can help. 

And yes I do plan adding a good readme with your own DB etc

**Comment by jimmykane** [comment]
If you need any help in 1:1 chat settings this up in your own firebase or local db we can find a way. There is a slack channel

**Comment by ryancornia** [comment]
Thanks, I will try to slack channel when I have some time to debug a bit more first. I noticed in the environments file:

  getGarminHealthAPIAuthRequestTokenRedirectURI: 'http://localhost:5000/quantified-self-io/europe-west2/getGarminHealthAPIAuthRequestTokenRedirectURI',
  
which made me think I needed another service running on localhost besides :4200.

**Comment by jimmykane** [comment]
Yeah I see. It's the functions. You could not run them for start. The
client should be able to run.

I need to add info also about how you can set and request an API token for
an app from SUUNTO and Garmin. That's why just plain running the functions,
in many cases, might not work.

Give me a little time

**Comment by nikossyr** [comment]
Hi! I stambled upon the project today and it seems very interesting. I wanted to run it locally and connect it to a local db (for privacy reasons) but with no luck. Do you have a wiki for that? Or anything similar for that matter?
Thank you! Great work!
